index.html

```diff
@@ -233,22 +233,22 @@ <h3>Latest news</h3>
         <div class="js-home-carousel home-carousel">
 
           <!-- News box Start -->
-          <div class="news-box">
+	  <div class="news-box">
             <div class="news-box-title">          
-              <span>19/07/2024</span>
+              <span>22/08/2024</span>
             </div>
             <div class="news-box-content">
-              <p>Uyuni 2024.07 is now available with new products enabled, changed behaviour of repo-sync, Confidential Computing Attestation UI and more! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
+              <p>Uyuni 2024.08 is now available with configuration of AppStreams via Activation Keys, migration "prepare" command, mgrprxy support config and more! This is the last version to support the RPM based product before moving to a containers only release! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```

Comment:
```suggestion
              <p>Uyuni 2024.08 is now available with configuration of AppStreams via Activation Keys, migration "prepare" command, mgrprxy support config and more! This is the last version to support the RPM based product before moving to a containers only release! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details and for a very important known issue!</p>
```
And maybe adjust the same at `news.html`

Reply:
Fixed in both places.